addons/xeh/fnc_postInit.sqf

```diff
@@ -19,6 +19,28 @@ Author:
 isNil {
     XEH_LOG("PostInit started. " + PFORMAT_9("MISSIONINIT",missionName,missionVersion,worldName,isMultiplayer,isServer,isDedicated,CBA_isHeadlessClient,hasInterface,didJIP));
 
+    addMissionEventHandler ["EntityCreated", {
+        params ["_entity"];
+        if ((!ISPROCESSED(_entity)) && (ISINCOMP(typeof _entity))) then {
+            _entity call CBA_fnc_initEvents;
+
+            if !(ISINITIALIZED(_entity)) then {
+                _entity call CBA_fnc_init;
+            };
+        };
+    }];
+    
+    // init object incompatile XEH
+    {
+        if !(ISPROCESSED(_x)) then {
+            _x call CBA_fnc_initEvents;
+
+            if !(ISINITIALIZED(_x)) then {
+                _x call CBA_fnc_init;
+            };
+        };
+    } forEach (entities [(call (uiNamespace getVariable [QGVAR(incompatibleClasses), {[]}])), [], true, true]);
```

Comment:
There's no point in adding the EH if there's no incompatible classes. Also if we would add this in preInit there would be no need to run the forEach loop.
```suggestion
    private _incompatibleClasses = call (uiNamespace getVariable [QGVAR(incompatibleClasses), {[]}]);
    if (count _incompatibleClasses > 0) then {

        addMissionEventHandler ["EntityCreated", {
            params ["_entity"];
            if ((!ISPROCESSED(_entity)) && (ISINCOMP(typeOf _entity))) then {
                _entity call CBA_fnc_initEvents;

                if !(ISINITIALIZED(_entity)) then {
                    _entity call CBA_fnc_init;
                };
            };
        }];
        
        // init object incompatile XEH
        {
            if !(ISPROCESSED(_x)) then {
                _x call CBA_fnc_initEvents;

                if !(ISINITIALIZED(_x)) then {
                    _x call CBA_fnc_init;
                };
            };
        } forEach (entities [_incompatibleClasses, [], true, true]);
    };
```

Reply:
Yes its good idea